### ISSUE TITLE
Multiple issues with selects in fixed navbars

### ISSUE THREAD

**Issue by tsauvajon** [vuetifyjs/vuetify#1832]
### Steps to reproduce
Create an application with a `fixed` toolbar and a select in it.
Add a `prepend icon`, and the `solo` and `autocomplete` properties to the select

### Versions
Latest versions as of today (21/09/2019):
Vue v2.4.4
Vuetify v0.15.7

### What is expected ?
I expect the select items to be aligned with the text field (x-axis) and to stay just under the search box (y-axis)

### What is actually happening ?
- There is an offset on the left, corresponding to the prepend icon
- When I scroll through the page, the items scroll when they shouldn't : the search box is fixed (check the codepen)
- I can't reproduce it on codepen but on my actual application, the box opens far under my search box : ![Screenshot from the actual application](https://i.imgur.com/LiYrsBY.png) . You can see the x-axis offset and the y-axis offset.

### Reproduction Link
https://codepen.io/tsauvajon/pen/KXMxyv

### Further note
I can't seem to find the different slots available for selects. In the [documentation](https://vuetifyjs.com/components/selects#example-6), there are exemples for the "item" and "selection" slots. It would be nice to have a list of all avaible slots, and a slot to "customize" the dropdown menu (e.g. nudge-left, position-x properties)

**Comment by johnleider** [comment]
The toolbar sets the margin of the first and last elements to ensure proper spacing. When the select is the only element, it is getting these values. Temporary fix is to add the class **ma-0** to v-select.

**Comment by tsauvajon** [comment]
Thanks for the fast answer !
I've updated the pen, and added ma-0. It works well for simple layouts : the codepen is working as planned as long as the search bar isn't contained in a div with margins.

In my actual application though it did help at all :/

I added a css property to menus to temporarly fix it, it is 'ok' for high resolutions but looks quite off for lower resolutions (i.e. mobile)

```
.menu__content.menu__content--select.menu__content--dropdown.menu__content--autocomplete {
  position: fixed;
}
```
